docs/helm-chart.md

```diff
@@ -0,0 +1,77 @@
+# Setting up to use the Helm Chart
+
+The Helm Chart that this project is currently using is the Backstage Helm Chart. This Helm Chart is used to simplify the process of deploying the Backstage Showcase app to a Kubernetes cluster. For me information on the Backstage Helm Chart consult the [official documentation](https://github.com/backstage/charts).
+
+## Getting Started
+
+Ensure that you have Helm installed on your local machine, you can find out more information on the [Helm website](https://helm.sh/docs/intro/install/).
+
+Next, is to add the Backstage Helm Chart and its dependent repository.
+
+```shell
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add backstage https://backstage.github.io/charts
+```
+
+## Deploying to a Kubernetes cluster
+
+This guide will have a focus deploying to an OpenShift cluster as that is the main target for this project.
+
+Build and push an image of the Backstage Showcase app that you can test.
```

Comment:
Should this be an ordered list?

1. Build and push an image of the Backstage Showcase app that you can test.
   ...
2. Create a file to hold environment variables called values-backstage-showcase.yaml and populate it with the following content.
   ...
3. Deploy the Backstage Showcase app to the OpenShift cluster

etc